packages/project-portal-content-netlify/src/cms/cms.js

```diff
@@ -7,7 +7,7 @@ import "./cms-utils"
 // (if it exists in the site's directory static/admin/config.yml)
 CMS.init({
   config: {
-    media_folder: "content/image",
```

Comment:
This works great! Unfortunately, it breaks the display of images in other parts of the CMS – on the CardPage and Other Page config.
<img width="1447" alt="Screenshot 2023-10-20 at 14 20 28" src="https://github.com/thepolicylab-projectportals/project-portal-theme/assets/2803227/7c2489cc-259f-4419-86cd-00237967b9d6">

There are two ways I can see we could fix this: 
- add the line 
  ```
  media_folder: "/content/image",
  ```
  to the "card page" and "other pages" collections

- leave the `media_folder: "content/image",` here unchanged and modify the `media_folder` property of the "project" collection to be `media_folder: "/static/image",` (the leading slash is required, otherwise it gets treated as a relative path)